src/button/Button.ts

```diff
@@ -0,0 +1,66 @@
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { ThemeableMixin, ThemeableProperties, theme } from '@dojo/widget-core/mixins/Themeable';
+import { v } from '@dojo/widget-core/d';
+import * as css from './styles/button.css';
+
+/**
+ * @type ButtonProperties
+ *
+ * Properties that can be set on a Button component
+ *
+ * @property content			Text content of button
+ * @property describedBy	ID of element with descriptive text
+ * @property disabled			Whether the button is disabled or clickable
+ * @property hasPopup			Whether the button triggers a popup
+ * @property icon					Creates an icon button with the specified icon glyph
+ * @property name					The button's name attribute
+ * @property pressed			Indicates status of a toggle button
+ * @property type					Button type can be "submit", "reset", "button", or "menu"
+ * @property onClick			Called when the button is clicked
+ */
+export interface ButtonProperties extends ThemeableProperties {
+	content?: string;
```

Comment:
I've been trying to get rid of label as a keyword for text content in favor of label as a keyword for labels